src/transformers/configuration_utils.py

```diff
@@ -21,6 +21,8 @@
 import os
 from typing import Any, Dict, Tuple, Union
 
+from transformers import __version__
```

Comment:
Relative import would be better here
```suggestion
from . import __version__
```